staging/src/k8s.io/apiserver/pkg/server/filters/reqmgmt.go

```diff
@@ -255,17 +66,16 @@ func WithRequestManagementByClient(
 		}
 
 		for {
-			rmState := reqMgmt.curState.Load().(*RMState)
-			fs := reqMgmt.pickFlowSchema(r, rmState.flowSchemas, rmState.priorityLevelStates)
-			ps := reqMgmt.requestPriorityState(r, fs, rmState.priorityLevelStates)
-			if ps.config.Exempt {
+			rmState := reqMgmt.GetCurrentState()
+			fs := utilflowcontrol.PickFlowSchema(requestDigest, rmState.GetFlowSchemas(), rmState.GetPriorityLevelStates())
+			ps := utilflowcontrol.RequestPriorityState(requestDigest, fs, rmState.GetPriorityLevelStates())
+			if ps.IsExempt() {
 				klog.V(5).Infof("Serving %v without delay\n", r)
 				handler.ServeHTTP(w, r)
 				return
 			}
-			flowDistinguisher := reqMgmt.computeFlowDistinguisher(r, fs)
-			hashValue := reqMgmt.hashFlowID(fs.Name, flowDistinguisher)
-			quiescent, execute, afterExecute := ps.fqs.Wait(hashValue, ps.config.HandSize)
+			hashValue := utilflowcontrol.ComputeFlowDistinguisher(requestDigest, fs)
```

Comment:
That function is now misnamed; it does not return a flow distinguisher, it returns a hash value.

Reply:
is it also updated in #80956?

Reply:
Indeed.